target/linux/ramips/base-files/etc/hotplug.d/firmware/10-rt2x00-eeprom

```diff
@@ -71,6 +71,11 @@ case "$FIRMWARE" in
 		rt2x00_eeprom_extract "factory" 0x0 0x200
 		rt2x00_eeprom_set_macaddr $wifi_mac
 		;;
+	elecom,wrc-733ghbk)
+		wifi_mac=$(mtd_get_mac_ascii hwconfig HW.LAN.2G.0.MAC.Address)
+		rt2x00_eeprom_extract "factory" 0x0 0x200
+		rt2x00_eeprom_set_macaddr $wifi_mac
```

Comment:
Hi, since you are using rt2x00_eeprom function here, I wanted to kindly ask you whether you could test my patchset for caldata extraction unification in PR #2464 on this device.

You would just need the first four patches in the PR and needed to change this code to the following:

```
elecom,wrc-733ghbk)
	wifi_mac=$(mtd_get_mac_ascii hwconfig HW.LAN.2G.0.MAC.Address)
	rt2x00_eeprom_extract "factory" 0x0 0x200
	caldata_patch_fw_mac $wifi_mac 0x4
```

Don't forget to add the 0x4 in the last line.

I'm just asking for a favor, so feel free to decline this request and just mark it as resolved then.

Reply:
The referenced patchset has been merged, so this can be marked as resolved.